upload/src/addons/SV/ThreadReplyBanner/Entity/ThreadBanner.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace SV\ThreadReplyBanner\Entity;
+
+use XF\Mvc\Entity\Entity;
+use XF\Mvc\Entity\Structure;
+
+class ThreadBanner extends Entity {

```

Comment:
Use `cmd.php xf-dev:entity-class-properties` CLI to add class properties to entities. Including extended ones